index.html

```diff
@@ -14,4 +14,10 @@ <h2>Services available</h2>
         <li>service 3</li>
         <li>service 4</li>
     </ul>
+
+    <footer>
+        <li>Copyright 2023</li>
+        <li>This is our twitter</li>
```

Comment:
Maybe you can also add links to those social media accounts, so that users can be redirected if they want to check them.